backend/src/test/java/heartbeat/service/jira/JiraBoardConfigDTOFixture.java

```diff
@@ -64,6 +64,10 @@ public class JiraBoardConfigDTOFixture {
 
 	public static final String ASSIGNEE_NAME = "Zhang San";
 
+	public static final String DISPLAY_NAME_ONE = "da pei";
```

Comment:
what is `da pei` and `xiao pei`

Reply:
public static final String DISPLAY_NAME_ONE = "da pei"; ->public static final String DISPLAY_NAME_ONE = "Display name";

.....2

Reply:
chenge the name to common pattern